x-pack/legacy/plugins/remote_clusters/public/app/sections/components/remote_cluster_form/remote_cluster_form.js

```diff
@@ -516,57 +518,69 @@ export class RemoteClusterForm extends Component {
       return null;
     }
 
-    const errorExplanation = [];
+    const errorExplanations = [];
 
     if (errorClusterName) {
-      errorExplanation.push({
+      errorExplanations.push({
         key: 'nameExplanation',
-        where: i18n.translate('xpack.remoteClusters.remoteClusterForm.inputNameErrorsScreenReaderExplanation', {
-          defaultMessage: 'The input "Name". You have entered: {name}.',
-          values: { name }
+        field: i18n.translate('xpack.remoteClusters.remoteClusterForm.inputNameErrorMessage', {
+          defaultMessage: 'The "Name" field is invalid.',
+        }),
+        error: errorClusterName
+      });
+    }
+
+    if (errorsSeeds) {
+      errorExplanations.push({
+        key: 'seedsExplanation',
+        field: i18n.translate('xpack.remoteClusters.remoteClusterForm.inputSeedsErrorMessage', {
+          defaultMessage: 'The "Seed nodes" field is invalid.',
         }),
-        what: errorClusterName
+        error: errorsSeeds
       });
     }
+
     if (localSeedErrors && localSeedErrors.length) {
-      errorExplanation.push({
-        key: 'seedExplanation',
-        where: i18n.translate('xpack.remoteClusters.remoteClusterForm.inputSeedErrorsScreenReaderExplanation', {
-          defaultMessage: 'The "Seed Nodes" input. You have entered: {seedInput}.',
-          values: { seedInput }
+      errorExplanations.push({
+        key: 'localSeedExplanation',
+        field: i18n.translate('xpack.remoteClusters.remoteClusterForm.inputLocalSeedErrorMessage', {
+          defaultMessage: 'The "Seed nodes" field is invalid.',
         }),
-        what: localSeedErrors.join(' ')
+        error: localSeedErrors.join(' '),
       });
     }
 
-    const messagesToBeRendered = (
-      errorExplanation.length && errorExplanation.map(errorMessage => (
-        <EuiScreenReaderOnly key={errorMessage.key}>
-          <p>
-            {errorMessage.where}<br/>
-            {errorMessage.what}
-          </p>
-        </EuiScreenReaderOnly>
-      ))
+    const messagesToBeRendered = errorExplanations.length && (
+      <EuiScreenReaderOnly>
+        <dl id={this.state.generateId(ERROR_LIST_ID)} aria-labelledby={this.state.generateId(ERROR_TITLE_ID)}>
+          {errorExplanations.map(({ key, field, error }) => (
+            <div key={key}>
+              <dt>{field}</dt>
+              <dd>{error}</dd>
+            </div>
+          ))}
+        </dl>
+      </EuiScreenReaderOnly>
     );
 
     return (
       <Fragment>
         <EuiSpacer size="m" data-test-subj="remoteClusterFormGlobalError" />
         <EuiCallOut
           title={(
-            <FormattedMessage
-              id="xpack.remoteClusters.remoteClusterForm.errorTitle"
-              defaultMessage="Fix errors before continuing."
-            />
+            <h2 id={this.state.generateId(ERROR_TITLE_ID)}>
```

Comment:
Great idea making this an `h2`!

I noticed that there are `h4` elements on this page, but no `h3`. Would you mind fixing this hierarchy gap by changing the `h4` elements to `h3` elements?

Reply:
Given that the form is always on the page and the errors are kind of secondary to the form, I'm thinking I change the form headers to `h2` and the error header to an `h3`. Thoughts?

Reply:
Sounds good!